### PR TITLE
Tweaking instructions

### LINKED ISSUE #29: Can't apply suggested change
## Bug Report

**Current behavior**
The instructions here: https://github.com/githubtraining/intro-to-html/blob/master/responses/06-add-image.md suggest applying the suggested change as an option, but it's not actually possible.

Screen Shot 2020-02-04 at 1 52 46 PM

**Reproduction**
Steps to reproduce the behavior in the course:
1. Take the course through step 6.
2. Click on 'Commit suggestion' in step 7.

**Expected behavior**
I would expect it not to mention applying the suggested change if it's not possible.

**Possible solution**
Change `Apply this suggested change, or follow the instructions below if you'd like to type out the code manually.` to `Follow the instructions below to add an image.  You can use the image suggested below or link to your image of choice.`

**Additional context**
PR to follow.

/cc @crichID @brianamarie @hectorsector @JasonEtco

### PR DESCRIPTION
**Pull request by amyschoen**
Applying a suggestion on a deleted line is not possible.

Closes #29 

/cc @crichID @brianamarie @hectorsector @JasonEtco

### CODE REVIEW

File: responses/06-add-image.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ You'll also notice that the image tag doesn't need to close since it is consider
 
 ### :keyboard: Activity: Add an image to your web page
 
-Apply this suggested change, or follow the instructions below if you'd like to type out the code manually.
+Follow the instructions below to add an image.  You can use the image suggested below or link to your image of choice.
```
Comment (crichID) [inline responses/06-add-image.md]:
```suggestion
Follow the instructions below to add an image.  You can use your GitHub avatar image (suggested below) or link to your image of choice.
```

**Comment by amyschoen** [comment]
Not a bad thought @crichID.  I stuck with what generically applied to my version locally and the one here.  The current release of LL uses profile pic instead of Octocat.

**Comment by brianamarie** [comment]
I see your point @crichID, that the more context, the better. But I also see your point @amyschoen that the current instructions are not as universal for other GHES courses.

I am personally in favor of the less specific instructions.


> Applying a suggestion on a deleted line is not possible.

This is not actually addressed yet in this pull request. The suggested change code [happens in the config](https://github.com/amyschoen/intro-to-html/blob/patch-1/config.yml#L183-L184). @amyschoen Would there be another way to edit that so the suggested change functions as expected?

**Comment by amyschoen** [comment]
Suggestions are new, so I was wondering how that tied in.  I was just experiencing the behavior where it had a suggestion that wasn't able to be applied.  Should I play around with changing the line number in the config to see if the suggestion could potentially work?  I somehow missed uploading the screenshot when I filed this.  Here's what it looks like currently:

Screen Shot 2020-02-04 at 1 49 56 PM

You can see the `Commit suggestion` button is grayed out.  The message if you click it anyway is more or less the first line of the PR details.

**Comment by brianamarie** [comment]
> Should I play around with changing the line number in the config to see if the suggestion could potentially work?

👍 Yes please, this would be ideal. Thank you!

**Comment by amyschoen** [comment]
I'm planning on doing a bunch more testing today.  Do you have any documentation you can point me towards that would help?  An informal ping on Slack with the info I need will suffice if there's not formal documentation available.

**Comment by brianamarie** [comment]
> I'm planning on doing a bunch more testing today. Do you have any documentation you can point me towards that would help? An informal ping on Slack with the info I need will suffice if there's not formal documentation available.

I pinged you in Slack, but I'll post here too in case anyone else could find this useful:
- https://lab.github.com/githubtraining/write-a-learning-lab-course/
- https://lab.github.com/docs

**Comment by amyschoen** [comment]
Thanks for the links @brianamarie.  This is way easier to use as a reference than scrolling up in Slack history.

**Comment by amyschoen** [comment]
Closing and opening up #35 instead because I deleted my fork too early.

**Pull request closed without merging**